mx_commands.py

```diff
@@ -125,6 +125,9 @@ def __init__(self, mx_commands, command_function, suite_name, command, usage_msg
     def command_function(self):
         return self._command_function
 
+    def has_documentation(self):
+        return self.doc_function != None
```

Comment:
From the usage of `has_documentation`, it looks like this should also take the presence of a `usage_msg` into account.

Reply:
Ah, you're right. Actually, from the usage I think maybe the `has_documentation` function could be replaced by a `get_doc` function that creates the doc that is being used there:

```python
# mx.py
def help_(args):
    # ...
    print 'mx {0} {1}\n\n{2}\n'.format(name, command.usage_msg, command.get_doc())
```
Where:
```python
# MxCommand
def get_doc(self):
    doc = self.command_function.__doc__
    format_args = []
    if self.usage_msg:
        format_args.append(self.usage_msg)
    if self.doc_function:
        format_args.append(self.doc_function())
    return doc.format(*format_args)
```

What do you think?